pac4j-cas/pom.xml

```diff
@@ -33,6 +33,13 @@
 			<artifactId>pac4j-core</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>org.pac4j</groupId>
+			<artifactId>pac4j-http</artifactId>
+			<scope>compile</scope>
+			<optional>true</optional>
+			<version>${project.version}</version>
```

Comment:
The version should not be required here, this primary dependency should be defined in the parent `pom.xml` file.